tests/test_vectors.py

```diff
@@ -117,6 +118,13 @@ def _open(file):
     _equivs_file(u'timestampSuperfluousOffset.10n')  # TODO amzn/ion-python#121
 )
 
+
+if c_ext:
+    _SKIP_LIST += (
+        _good_file(u'subfieldVarInt.ion'),  # c_ext supports 300 decimal digits while there is a 8000+ decimal digits test..
```

Comment:
Need to figure out why this file is not in ion-c skip list.

Reply:
+1

Reply:
This is because we used string to convert between python decimal and ion decimal, it will be solved once we calculate them without using string.

Reply:
Our current algorithm of ion/python decimal conversion only support 34 digits. So we may still want to put it in skip list. 

Looked into ion-c tests, this is a good vector and ion-c read value as an event_stream. So the value is stored as a string in an ion_event and doesn't raise an error.